builder/pipeline.h

```diff
@@ -310,20 +311,26 @@ class func {
   }
 
   // Make a copy from a single input to a single output.
-  static func make_copy(input src, output dst) { return func({std::move(src)}, std::move(dst)); }
+  static func make_copy(input src, output dst, copy_stmt::callable impl = slinky::copy) {
```

Comment:
I think it'd be good to update the comment to mention what is impl for and what are the expectations.